src/pe_source/cybersixgill.py

```diff
@@ -0,0 +1,298 @@
+    alerts,
+    alias_organization,
+    creds,
+    cve_summary,
+    mentions,
+    root_domains,
+    top_cves,
+)
+
+# Set todays date formatted YYYY-MM-DD and the start_date 30 days prior
+TODAY = date.today()
+DAYS_BACK = timedelta(days=30)
+START_DATE = str(TODAY - DAYS_BACK)
+END_DATE = str(TODAY)
+DATE_SPAN = f"[{START_DATE} TO {END_DATE}]"
+
+# Set dates to YYY-MM-DD H:M:S format
+NOW = datetime.now()
+BACK = timedelta(days=30)
+FROM_DATE = (NOW - BACK).strftime("%Y-%m-%d %H:%M:%S")
+TO_DATE = NOW.strftime("%Y-%m-%d %H:%M:%S")
+
+
+class Cybersixgill:
+    """Fetch Cybersixgill data."""
+
+    def __init__(self, orgs_list, method_list):
+        """Initialize Cybersixgill class."""
+        self.orgs_list = orgs_list
+        self.method_list = method_list
+
+    def run_cybersixgill(self):
+        """Run Cybersixgill api calls."""
+        orgs_list = self.orgs_list
+        method_list = self.method_list
+
+        # Get org info from PE database
+        pe_orgs = get_orgs()
+
+        # Get Cybersixgill org info
+        sixgill_orgs = get_sixgill_organizations()
+        failed = []
+        count = 0
+
+        # Get data source uid
+        source_uid = get_data_source_uid("Cybersixgill")
+
+        # Run top CVEs. Same for all orgs
+        if "topCVEs" in method_list:
+            topCVEs = self.get_topCVEs(source_uid)
+            if topCVEs == 1:
+                failed.append("Top CVEs")
+
+        for pe_org in pe_orgs:
+            org_id = pe_org[2]
+            pe_org_uid = pe_org[0]
+            # Only run on specified orgs
+            if org_id in orgs_list or orgs_list == "all":
+                count += 1
+                # Get sixgill_org_id associated with the PE org
+                try:
+                    sixgill_org_id = sixgill_orgs[org_id][5]
+                except KeyError as err:
+                    logging.error("PE org is not listed in Cybersixgill.")
+                    print(err, file=sys.stderr)
+                    failed.append("%s not in sixgill" % org_id)
+                    continue
+
+                # Run alerts
+                if "alerts" in method_list:
+                    alert = self.get_alerts(
+                        org_id, sixgill_org_id, pe_org_uid, source_uid
+                    )
+                    if alert == 1:
+                        failed.append("%s alerts" % org_id)
+                # Run mentions
+                if "mentions" in method_list:
+                    mention = self.get_mentions(
+                        org_id, sixgill_org_id, pe_org_uid, source_uid
+                    )
+                    if mention == 1:
+                        failed.append("%s mentions" % org_id)
+                # Run credentials
+                if "credentials" in method_list:
+                    cred = self.get_credentials(
+                        org_id, sixgill_org_id, pe_org_uid, source_uid
+                    )
+                    if cred == 1:
```

Comment:
Remove unneeded variable:
```suggestion
                    if self.get_credentials(
                        org_id, sixgill_org_id, pe_org_uid, source_uid
                    ) == 1:
```